Documentation/getting-started.md

```diff
@@ -39,22 +40,36 @@ public class MobileApp extends Application {
     public void onCreate() {
       super.onCreate();
       MobileCore.setApplication(this);
-      try {
-        Edge.registerExtension();
-        Identity.registerExtension();
-        // register other extensions
-        MobileCore.start(new AdobeCallback () {
-            @Override
-            public void call(Object o) {
-                MobileCore.configureWithAppID("yourAppId");
-            }
-        });      
-
-      } catch (Exception e) {
-        ...
-      }
-
-
+    
+		  MobileCore.configureWithAppID(ENVIRONMENT_FILE_ID);
+
+		// register Adobe extensions
+		MobileCore.registerExtensions(
+			Arrays.asList(Edge.EXTENSION, Identity.EXTENSION, Assurance.EXTENSION),
+			o -> Log.d("MobileApp", "Mobile SDK was initialized")
+		);
+	}
+}
+```
+### Kotlin
+
+```kotlin
+class MobileApp : Application() {
+    // Set up the preferred Environment File ID from your mobile property configured in Data Collection UI
+    private var ENVIRONMENT_FILE_ID: String = ""
+
+    override fun onCreate() {
+        super.onCreate()
+
+        // register AEP SDK extensions
+        MobileCore.setApplication(this)
+        MobileCore.setLogLevel(LoggingMode.VERBOSE)
+        
+        MobileCore.registerExtensions(
+            listOf(Edge.EXTENSION, Identity.EXTENSION, Consent.EXTENSION, Assurance.EXTENSION)
```

Comment:
let's keep only the required sdks for the getting started guides, Edge and Identity